jina/clients/sugary_io.py

```diff
@@ -102,7 +102,7 @@ def _input_files(
         sampling_rate: float = None,
         read_mode: str = None,
 ) -> Iterator[Union[str, bytes]]:
-    """Input function that iterates over files, it can be used in the Flow API
+    r"""Input function that iterates over files, it can be used in the Flow API.
```

Comment:
What does the prefix `r` do here?

Reply:
The linter says to use `r`-strings when we have an escape character inside (the `\`).